WoWsShipBuilder.Web/Dialogs/CustomAccelerationDataDialog.razor

```diff
@@ -63,7 +63,7 @@
         <MudButton OnClick="Cancel" Variant="Variant.Filled" Color="Color.Primary">
             @Localizer.GetAppLocalization(nameof(Translation.Cancel)).Localization
         </MudButton>
-        <MudButton Disabled="@string.IsNullOrWhiteSpace(name)" OnClick="ReturnShipBuildContainer" Variant="Variant.Filled" Color="Color.Primary">
+        <MudButton Disabled="ValidateCustomParameters()" OnClick="ReturnShipBuildContainer" Variant="Variant.Filled" Color="Color.Primary">
```

Comment:
The name is misleading, the method name does not look like it is used to check whether a button is disabled